extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js

```diff
@@ -8,18 +9,25 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements', [
 	var logGroup = 'ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements',
 		placementsMap = instantGlobals.wgAdDriverAppNexusBidderPlacementsConfig;
 
-	function getPlacement(skin) {
+	function getPlacement(skin, position) {
 		var vertical = adContext.getContext().targeting.mappedVerticalName;
 
 		if (placementsMap && placementsMap[skin] && placementsMap[skin][vertical]) {
-			return placementsMap[skin][vertical];
-		} else {
-			log([
-					'getPlacement', skin, vertical,
-					'wgAdDriverAppNexusBidderPlacementsConfig variable not set or not configured correctly'
-				], 'error', logGroup
-			);
+			if (position && placementsMap[skin][vertical][position]) {
+				return placementsMap[skin][vertical][position];
+			}
+
+			// TODO: remove if statement after switching config from old structure (conf[skin][vertical] = pId)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1135)